samples/snippets/create_batches_sample.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START documentai_toolbox_create_batches]
+
+from google.cloud import documentai
+from google.cloud.documentai_toolbox.utilities import utilities
+
+# TODO(developer): Uncomment these variables before running the sample.
+# Given unprocessed documents in path gs://bucket/path/to/folder
+# gcs_bucket_name = "bucket"
+# gcs_prefix = "path/to/folder"
+# batch_size = 50
+
+
+def create_batches_sample(
```

Comment:
Nit: could you add another function that uses the batches to call documentai, i think it would be helpful to show how to use the batches.

Reply:
I considered this, but it would be a bunch of duplicated code from the batch_process sample. This will be embedded in the docs on the same page that shows how to send a batch process request. I'll add a link